include/bluetooth/conn.h

```diff
@@ -711,8 +711,78 @@ struct bt_conn_oob_info {
 	};
 };
 
+#if defined(CONFIG_BT_SMP_PAIRING_ALLOW)
+/* Info structure for generic_pairing_query */
+struct bt_conn_pairing_auth_req_info {
+	/** AuthReq, Core Spec. Vol 3, Part H, 3.5.1, Fig. 3.3 */
+	u8_t bonding_flags:2,
+	     mitm:1,
+	     sc:1,
+	     keypress:1,
+	     ct2:1,
+	     rfu:2;
+};
+
+/** @brief Pairing request and pairing response info structure.
+ *
+ *  This structure is the same for both smp_pairing_req and smp_pairing_rsp
+ *  and a subset of the packet data, except for the initial Code octet.
+ *  It is documented in Core Spec. Vol. 3, Part H, 3.5.1 and 3.5.2.
+ */
+struct bt_conn_pairing_info {
+	/** IO Capability, Core Spec. Vol 3, Part H, 3.5.1, Table 3.4 */
+	u8_t io_capability;
+	/** OOB data flag, Core Spec. Vol 3, Part H, 3.5.1, Table 3.5 */
+	u8_t oob_data_flag;
```

Comment:
For improved readability could you add an empty line after each variable. That makes it easier to see where the documentation for the previous variable ends and a new variable definition begins.